search/build.gradle

```diff
@@ -5,15 +5,3 @@
  * https://www.gnu.org/licenses/lgpl-3.0.html
  */
 /* Build Script */
-
-bootJar {
-    launchScript()
```

Comment:
❓ Why are we removing the docker and boot jar from search?

Reply:
I think we had it there so that the org.springframework.boot wouldn't complain that it didn't have a main class. Now that we moved that plugin to the sub modules that are actually spring boot, I don't think we need that stuff.

Reply:
I actually just realized that there is `@SpringBootApplication` in the search app. I will update the build.gradle to match the other sub-projects.